std_msgs/String topic
std_msgs/String server_namespace
---